.travis.yml

```diff
@@ -28,97 +28,98 @@ jobs:
-  - env: ARCH="macos32x86" FLAVOR="squeak.cog.spur"
+    env: ARCH="linux32x86"    FLAVOR="squeak.cog.spur"          TESTIMAGE="Squeak-5.1"
+  - env: ARCH="macos32x86"    FLAVOR="squeak.cog.spur"          TESTIMAGE="Squeak-5.1"
     <<: *mac-build
-  - env: ARCH="linux64x64" FLAVOR="squeak.cog.spur"
-  - env: ARCH="macos64x64" FLAVOR="squeak.cog.spur"
+  - env: ARCH="linux64x64"    FLAVOR="squeak.cog.spur"          TESTIMAGE="skip"
+  - env: ARCH="macos64x64"    FLAVOR="squeak.cog.spur"          TESTIMAGE="skip"
     <<: *mac-build
-  - env: ARCH="linux32x86" FLAVOR="pharo.cog.spur" HEARTBEAT="threaded"
-  - env: ARCH="macos32x86" FLAVOR="pharo.cog.spur"
+  - env: ARCH="linux32x86"    FLAVOR="pharo.cog.spur"           TESTIMAGE="Pharo-5.0"   HEARTBEAT="threaded"
+  - env: ARCH="macos32x86"    FLAVOR="pharo.cog.spur"           TESTIMAGE="Pharo-5.0"
     <<: *mac-build
-  - env: ARCH="linux64x64" FLAVOR="pharo.cog.spur" HEARTBEAT="threaded"
-  - env: ARCH="macos64x64" FLAVOR="pharo.cog.spur"
+  - env: ARCH="linux64x64"    FLAVOR="pharo.cog.spur"           TESTIMAGE="skip"        HEARTBEAT="threaded" 
+  - env: ARCH="macos64x64"    FLAVOR="pharo.cog.spur"           TESTIMAGE="skip"
     <<: *mac-build
 
   - stage: "Other Linux builds"
-    env: ARCH="linux64x64" FLAVOR="newspeak.cog.spur"
-  - env: ARCH="linux64x64" FLAVOR="pharo.cog.spur" HEARTBEAT="itimer"
-  # - env: ARCH="linux64x64" FLAVOR="pharo.sista.spur" HEARTBEAT="threaded"
-  - env: ARCH="linux32x86" FLAVOR="newspeak.cog.spur"
-  - env: ARCH="linux32x86" FLAVOR="squeak.cog.v3"
-  - env: ARCH="linux32x86" FLAVOR="squeak.sista.spur"
-  - env: ARCH="linux32x86" FLAVOR="pharo.cog.spur" HEARTBEAT="itimer"
-  - env: ARCH="linux32x86" FLAVOR="pharo.sista.spur" HEARTBEAT="threaded"
+    env: ARCH="linux64x64"    FLAVOR="newspeak.cog.spur"        TESTIMAGE="skip"
+  - env: ARCH="linux64x64"    FLAVOR="pharo.cog.spur"           TESTIMAGE="skip"        HEARTBEAT="itimer"
+  #-env: ARCH="linux64x64"    FLAVOR="pharo.sista.spur"         TESTIMAGE="skip"        HEARTBEAT="threaded"
+  - env: ARCH="linux32x86"    FLAVOR="newspeak.cog.spur"        TESTIMAGE="refer newspeakBootstrap.sh"
+  - env: ARCH="linux32x86"    FLAVOR="squeak.cog.v3"            TESTIMAGE="Squeak-4.6"
+  - env: ARCH="linux32x86"    FLAVOR="squeak.sista.spur"        TESTIMAGE="skip"
+  - env: ARCH="linux32x86"    FLAVOR="pharo.cog.spur"           TESTIMAGE="Pharo-5.0"   HEARTBEAT="itimer"
+  - env: ARCH="linux32x86"    FLAVOR="pharo.sista.spur"         TESTIMAGE="skip"        HEARTBEAT="threaded"
     compiler: clang
-  - env: ARCH="linux32x86" FLAVOR="pharo.sista.spur" HEARTBEAT="itimer" 
+  - env: ARCH="linux32x86"    FLAVOR="pharo.sista.spur"         TESTIMAGE="skip"        HEARTBEAT="itimer" 
     compiler: clang
 
   - stage: "Other Mac builds"
-    env: ARCH="macos64x64" FLAVOR="newspeak.cog.spur"
+    env: ARCH="macos64x64"    FLAVOR="newspeak.cog.spur"        TESTIMAGE="refer newspeakBootstrap.sh"
```

Comment:
I don't really have a better suggestion, so let's go with *refer newspeakBootstrap.sh* for now.